crates/biome_grit_formatter/src/grit/lists/definition_list.rs

```diff
@@ -5,6 +5,22 @@ pub(crate) struct FormatGritDefinitionList;
 impl FormatRule<GritDefinitionList> for FormatGritDefinitionList {
     type Context = GritFormatContext;
     fn fmt(&self, node: &GritDefinitionList, f: &mut GritFormatter) -> FormatResult<()> {
-        format_verbatim_node(node.syntax()).fmt(f)
+        let mut join: JoinNodesBuilder<'_, '_, Line, GritFormatContext> =
```

Comment:
```suggestion
        let mut join =
```
Usually, types are added only when the compiler can't infer them

Reply:
My VSCode is being stubborn, and keeps putting it in there whenever I delete it for some reason 😅